src/main/java/edu/cornell/kfs/pmw/batch/service/impl/PaymentWorksNewVendorRequestsServiceImpl.java

```diff
@@ -135,14 +119,14 @@ private boolean pmwNewVendorRequestProcessingIntoKfsWasSuccessful(PaymentWorksVe
     
     private boolean canPaymentWorksNewVendorRequestProcessingContinueForVendor(PaymentWorksVendor stgNewVendorRequestDetailToProcess, PaymentWorksNewVendorRequestsBatchReportData reportData) {
         List<String> errorMessages = new ArrayList<String>();
-        if (pmwDtosCouldConvertCustomAttributesToPmwJavaClassAttributes(stgNewVendorRequestDetailToProcess) &&
-            pmwNewVendorAttributesConformToKfsLengthsOrFormats(stgNewVendorRequestDetailToProcess, errorMessages) &&
-            allPmwNewVendorIsoCountriesMapToSingleFipsCountry(stgNewVendorRequestDetailToProcess, errorMessages) &&
-            pmwNewVendorIdentifierDoesNotExistInKfsStagingTable(stgNewVendorRequestDetailToProcess, errorMessages)){
+        if (ObjectUtils.isNotNull(stgNewVendorRequestDetailToProcess)
```

Comment:
It looks like that if the vendor object is null, then the report data might not get updated to indicate that the vendor could not be processed. I think the report needs to be updated accordingly when the data for a specific vendor could not be retrieved. If necessary, such reporting could be added to the "obtainPmwNewVendorRequestDetailForPmwIdentifier" method instead.

Reply:
I think it makes sense to add this report logging here, I will extract the null check into a function, and when null, add to the error List